pkgs/tools/misc/panoply/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, stdenvNoCC, fetchurl, makeWrapper, jre } :
+
+stdenvNoCC.mkDerivation rec {
+  pname = "panoply";
+  version = "5.0.5";
+
+  src = fetchurl {
+    url = "https://www.giss.nasa.gov/tools/panoply/download/PanoplyJ-${version}.tgz";
+    sha256 = "0yzqm6c2sgfxyijb5ywx7m1dwv4nwhccfs54wyqz2hnjl8xw34r3";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
+    cp panoply.sh $out/bin/panoply
+    cp -r jars $out/bin
```

Comment:
Does this really add a jars command?

Reply:
It is a sub directory which contains the java files. The startup script expects it in the same directory. It is possible to put it somewhere else but then the script needs to be patched, adding more complexity.

Reply:
We are not going to put a jars directory into $out/bin for a tool. Please move this to share or similar and add a wrapper.